eng/pipelines/libraries/outerloop.yml

```diff
@@ -72,12 +72,12 @@ jobs:
         buildConfig: Debug
         platforms:
         - ${{ if eq(variables['includeWindowsOuterloop'], true) }}:
-          - Windows_NT_x64
+          - win_x64
         - ${{ if eq(variables['includeLinuxOuterloop'], true) }}:
```

Comment:
Would this need to be lowercase to match other changes - 'includelinuxOuterloop'?

Curious why only Linux-one was being changed to lowercase and not Windows and Osx?